src/Components/Blazor/Build/src/ReferenceFromSource.props

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <RunCommand>dotnet</RunCommand>
     <_BlazorCliLocation>$(MSBuildThisFileDirectory)../../DevServer/src/bin/$(Configuration)/netcoreapp3.0/blazor-devserver.dll</_BlazorCliLocation>
-    <RunArguments>exec &quot;$(_BlazorCliLocation)&quot; serve $(AdditionalRunArguments)</RunArguments>
+    <RunArguments>exec &quot;$(_BlazorCliLocation)&quot; serve $(OutputPath)$(TargetFileName) $(AdditionalRunArguments)</RunArguments>
```

Comment:
This also improves the DevServer scenario and will fix a bug users have been having with Windows 7. I chose to include this fix because it actually causes issues that are hard to diagnose when running the functional E2E tests

Reply:
That's cool. Looks like it will also fix https://github.com/aspnet/AspNetCore/issues/5486, right?

Might it be worth passing the absolute path to the binary here though? I haven't tried this to check it, but I'd guess if you did `dotnet run --project someotherpath` then this would fail because `$(OutputPath)` would be relative to the `.csproj`, but at runtime it would try to resolve it relative to the CWD. Or am I misreading this?

Reply:
Also I think the `$(OutputPath)$(TargetFileName)` part should be surrounded in `&quot;` since either part of that could contain a space.

Reply:
Yes, and the windows 7 issue. And an issue where the E2E tests don't run proper if you built from command-line (which is why I ended up fixing it, cuz it cost me 1h of my life)

Reply:
You might be right about relative/full, but that's an easy fix.

Reply:
I did add the full path

Reply:
and validated calling `dotnet run --project projectfilepath`